### DISCUSSION TITLE
Welcome to nCine Discussions!

### DISCUSSION THREAD

**Discussion by encelo** [nCine/nCine#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the nCine community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Fahien** [comment]
Hi, I have been following the development of this project for some years, and I guess I can say I am one of the first nCine beta-testers!
Anyway, I try to be helpful when I can and I have some spare time.

**Reply by encelo** [reply]
Welcome here @Fahien. You're definitely one of the very first user, maybe the first! 😉

**Comment by akimky** [comment]
Hi! Are there plans to add IOS support?

**Reply by encelo** [reply]
Next big task, now that viewports and custom shaders are done, is probably going to be a Vulkan backend and a graphics API abstraction layer.
With this layer in place, writing a Metal backend for iOS and macOS would then be possible. 👍🏻
